contracts/contracts/vault/VaultCore.sol

```diff
@@ -576,7 +567,7 @@ contract VaultCore is VaultStorage {
         uint256 assetCount = getAssetCount();
         uint256[] memory assetPrices = _getAssetPrices();
         uint256[] memory assetBalances = new uint256[](assetCount);
-        uint256[] memory assetDecimals = new uint256[](assetCount);
+        uint256[] memory assetUnits = new uint256[](assetCount);
```

Comment:
since this var is used only inside the for loop we can remove it as a local variable inside the function